create.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App Create</title>
+    <link rel="stylesheet" href="./styles/index.css" />
+  </head>
+  <body>
+    <h1>Create</h1>
+    <main>
+      <form aria-labelledby="create-question-heading" autocoplete="off">
```

Comment:
😊 little typo..autocomplete

as i understood the wireframe it is not necessary to give a new question a heading , so you could shorten the aria-labelledby value to "create-question"

Reply:
wow thank you Tanja, you are really looking into details - great 👍🏼